### DISCUSSION TITLE
CLI tool for nextjs authentication

### DISCUSSION THREAD

**Discussion by iamtouha** [saasykits/nextjs-sessionauth-template#51] [Ideas]
What are your thoughts on a new CLI tool designed to seamlessly implement authentication in any Next.js project within minutes? 

It would ideally include various authentication methods such as credential authentication, email link authentication, OTP authentication, etc. The workflow will be similar to Shadcn UI, allowing developers to easily customize the code to suit their specific requirements. 

Please share your thoughts and ideas about this project.

**Comment by Fr3akyMurk** [comment]
So you want to make a new project where you can do something like ``npx add-lucia-auth`` in order to easily add auth to any next.js project?
Sounds interesting but that would take a lot of time and effort.

**Reply by iamtouha** [reply]
yeah. That's why I was planning to start with barebone and add features incrementally. Like starting with drizzle and then adding prisma support later.
